owncloud/web
# Too low contrast on some UI elements

## Description
How to check contrast with Firefox:
2. login at https://ocis.ocis-web.latest.owncloud.works/
3. inspect site with Firefox Dev Tools > Tab "Accessibility" > check for issues "Contrast"
4. Select one of the found contrast violations
5. usually contrast should be chosen 4.5:1 (learn more: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast)

## Related Issue
- [Fixes #7353 ](https://github.com/owncloud/web/issues/7353)

## Motivation and Context


## How Has This Been Tested?



- test environment: Firefox Dev Tools
- inspect site with Firefox Dev Tools > Tab "Accessibility" > check for issues "Contrast"
- Select one of the found contrast violations
- ...

## Screenshot
![Group 1](https://user-images.githubusercontent.com/22890925/184843135-22bf9da7-4504-420e-9534-fd3ae2a06b17.png)
![Group 2](https://user-images.githubusercontent.com/22890925/184843289-77b0e158-c164-40e5-92e8-ef2279925c26.png)
![Group 3](https://user-images.githubusercontent.com/22890925/184843354-37976cc7-530c-4902-ac01-a6e8212cdca3.png)
![Group 4](https://user-images.githubusercontent.com/22890925/184843391-dedd0a6d-638a-45d6-8047-e46903f264c9.png)
![Group 5](https://user-images.githubusercontent.com/22890925/184843436-ecde59bf-9c50-445a-84d6-8b60587ba2a0.png)
![Group 6](https://user-images.githubusercontent.com/22890925/184843496-49d1cb56-ca57-41ac-9f5d-f5eb68520da2.png)
![Group 7](https://user-images.githubusercontent.com/22890925/184843532-64fcda19-6c56-4e9b-9d60-e24366b0f093.png)
![Group 8](https://user-images.githubusercontent.com/22890925/184843580-d99ea040-2b9d-4ba7-b89d-34850098efb1.png)

## Demo
[https://www.loom.com/share/3ae629035ed045ff805c8f56bdc41d2c](https://www.loom.com/share/3ae629035ed045ff805c8f56bdc41d2c
)

---

This PR was pushed through [GitStart](https://www.gitstart.com/), with contributions from @MJanuary @WajihaNiazi, @hashimaziz46
